# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../tre/tre-0.8.8-pkgcfg.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

 tre.pc.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/tre.pc.in b/tre.pc.in
index fdc45fa..b8d0676 100644
--- ./tre.pc.in
+++ ./tre.pc.in
@@ -6,5 +6,5 @@ includedir=@includedir@
 Name: TRE
 Description: TRE regexp matching library
 Version: @PACKAGE_VERSION@
-Libs: -L${libdir} -ltre @LDFLAGS@ @LIBINTL@ @LIBS@
+Libs: -L${libdir} -ltre @LIBINTL@ @LIBS@
 Cflags: -I${includedir} @CPPFLAGS@
